### ISSUE TITLE
/re top gui is broken

### ISSUE THREAD

**Issue by Joe7670** [VolmitSoftware/React#536]
**Describe the bug**
Using the command /re top opens up an inventory gui which is meant to order chunks in order of lag severity, showing details such as ms lag by category of lag. Currently, the feature is broken and doesn't show the details of the chunks, but clicking it still works and teleports you to the chunk

**To Reproduce**
Type /re top

**Expected behavior**
Using the command /re top opens up an inventory gui which is meant to order chunks in order of lag severity, showing details such as ms lag by category of lag.

**Server and Plugin Informations**
 - React Version: 6.618
 - paperspigot 1.15.2 git-paper-123

**Comment by snurre0** [comment]
I just want to add, that this is still a bug in paper-143.
If I remember correctly, It stopped working after a paper update, then it started working again at a later update, but then became broken again after yet another paper update, and has been broken since.

And just for a time/version reference, if it helps at all:
I found my old message in your discord now, and it seems that I asked you about this being broken at January 23rd.
Then I posted again to inform that it was fixed after we updated to paper-79.
Not sure in what version it broke again, but it was not too long after, and it's been broken since.
